### PR TITLE
Fix ipfs attempt2

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

Continues on last PR attempt to fix IPFS deployments (https://github.com/cowprotocol/cowswap/pull/4397)

As you can see, the previous PR fixed: The build and the deployment to IPFS 
https://github.com/cowprotocol/cowswap/actions/runs/8928121421

However, the website was not loading. This was because it was using absolute paths. I fought a lot with vite to make it work, and it was driving me nuts why although using a relative path approach, the app was trying to fetch using an absolute one (which ofcourse don't work in IPFS gateways)

At the end, I saw it had nothing to do with vite. It was just a freaking line in the HTML:
`<base href="/"/>`

## Test
🚨 The code I removed came from https://github.com/cowprotocol/cowswap/pull/3025, so we should make sure I didn't break anytning

For this reason, i bring the tests from the other PR here to repeat

TEST

1. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app
2. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app/
3. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app/?
4. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app/?utm_source=github
5. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app?utm_source=github
6. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app?other=parameter
7. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app?utm_source=github&other=parameter
8. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app/#/
8. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app/#/swap
9. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app/#/swap?utm_source=github
10. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app/#/swap?utm_source=github&other=parameter
11. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app/#/5/swap/WETH?utm_source=github
12. https://swap-dev-git-bring-ipfs-back-cowswap.vercel.app/#/5/swap/WETH?utm_source=github&other=parameter

### CODE REVIEW

**Comment by anxolin** [comment]
Merging as there's no-one working right now, I need to test the de generation of IPFS and I don't plan to deploy this to production. So, let me know if anyone is concerned with the changes

File: apps/cowswap-frontend/index.html [hunk 1]
```diff
@@ -8,7 +8,6 @@
       name="description"
       content="CoW Swap finds the lowest prices from all decentralized exchanges and DEX aggregators & saves you more with p2p trading and protection from MEV"
     />
-    <base href="/"/>
```
Comment (alfetopito) [inline apps/cowswap-frontend/index.html:11]: Any clue why this was here?

File: package.json [hunk 2]
```diff
@@ -23,8 +23,7 @@
     "build:cowfi": "nx build cow-fi",
     "build:cosmos": "nx run cowswap-frontend:build:cosmos",
     "ci:build": "yarn run build:cowswap && yarn run build-widget",
-    "ipfs:build": "PUBLIC_URL=\".\" yarn build",
-    "ipfs:publish": "npx ipfs-deploy build -p pinata -O",
+    "ipfs": "nx run cowswap-frontend:ipfs",
```
Comment (alfetopito) [inline package.json:26]: From what I can tell, the `ipfs` command is just doing a regular `build`.
What about removing it completely in favor of the `build` command instead?

File: apps/cowswap-frontend/index.html (hunk 1 above)
Comment (anxolin) [reply apps/cowswap-frontend/index.html:11]: No idea, but drove me crazy. I hoped you or Sasha would know

File: package.json (hunk 2 above)
Comment (anxolin) [reply package.json:26]: no, it depends on build, but it runs `npx ipfs-deploy build/cowswap -p pinata -O`

**Pull request merged**